modules/channel-slack/README.md

```diff
@@ -0,0 +1,40 @@
+# Slack Channel
```

Comment:
I'd add this to the official docs instead of the README or it can be both but at least the docs. In the `Channels` section.

Reply:
Once everything is completed and merged, i'll update the documentation for all channels at the same time, in the meantime we can test it and see if something is missing